src/main/resources/application.yml

```
@@ -20,4 +23,15 @@ springdoc:
+  refresh-token-validity-in-seconds: 86400
+
+cloud:
+  aws:
+    s3:
+      bucket: ${BUCKET_ADDRESS}
+    stack.auto: false
+    credentials:
+      accessKey: ${S3_ACCESS_KEY}
+      secretKey: ${S3_SECRET_KEY}
+    region:
+      _static: ap-northeast-2
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)